dissect/target/plugins/os/windows/log/evtx.py

```diff
@@ -169,3 +180,20 @@ def format_value(value: Any) -> Any:
         return utils.to_str(value)
     except UnicodeDecodeError:
         return repr(value)
+
+
+def unique_key(key: str, dictionary: dict[str, Any], count: int | None = None) -> str:
+    """Return a unique key for a given dict of key value pairs.
+
+    Makes the returned key unique by appending an incrementing integer after the given key name (e.g. ``key_2``).
+    Search is case sensitive so provide lower-cased ``key`` and ``dictionary`` arguments if case-insensitiveness
+    is desired.
+    """
+    count = count or 2
+    new_key = f"{key}_{count}"
```

Comment:
My only concern is that this behavior is exactly the same as with lists, maybe choose something different? Just adding an incremental amount of underscores? 😂

Reply:
Or put `duplicate` behind the digit? Maybe a little less confusing. I don't have a super strong opinion on it, just trying to think of something to avoid confusion.